java/kotlin-extractor/src/main/kotlin/utils/versions/v_1_5_0/IrLazyFunction.kt

```diff
@@ -0,0 +1,3 @@
+package com.github.codeql.utils.versions
```

Comment:
We can move the necessary bits from v_1_5* into `v_1_6_0`, and delete the rest, now we have dropped 1.5 support, but that can be done in a follow-up PR.